charts/bindplane/templates/bindplane-jobs.yaml

```diff
@@ -0,0 +1,447 @@
+{{- if eq (include "bindplane.deployment_type" .) "Deployment" }}
```

Comment:
This deployment is based on bindplane.yaml, with anything related to statefulset removed, such as volumes or Prometheus sidecar.